_translator-files/po/zh_CN/Software-Manual.po

```diff
@@ -91,364 +94,368 @@ msgid ""
 "\t<figcaption>Your local mix when connected to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"<img src=\"{% include img/en-screenshots/main-screen-medium.inc %}\" style=\"border: 5px solid gray;\" loading=\"lazy\" alt=\"Jamulus 主窗口的图像\">\n"
```

Comment:
Sorry, I overlooked this - you need to change the language code in `en-screenshot` to `zh_CN-screenshot`. That's for all screenshots.

Reply:
Yes, it will. The zh_CN folder for screenshots contains the ones in EN until they are replaced.

Reply:
done